app/components/primer/avatar_stack_component.rb

```diff
@@ -12,7 +12,7 @@ class AvatarStackComponent < Primer::Component
     TAG_OPTIONS = [DEFAULT_TAG, :span].freeze
 
     DEFAULT_BODY_TAG = :div
-    BODY_TAG_OPTIONS = TAG_OPTIONS = [DEFAULT_TAG, :span].freeze
+    BODY_TAG_OPTIONS = [DEFAULT_BODY_TAG, :span].freeze
```

Comment:
Is this not a functional change?

Reply:
or, hm, not it's not since `DEFAULT_TAG` == `DEFAULT_BODY_TAG`. We need both?

Reply:
no, the arrays end up being the same